### ISSUE TITLE
Whitespace in inserted textareas

### ISSUE THREAD

**Issue by jmuheim** [nathanvda/cocoon#323]
There's always some whitespace in textareas that were inserted using cocoon. How can I prevent this?

```html
<textarea class="text optional form-control pastable" placeholder="asdf" name="boilerplate[findings_attributes][1444981219987][screenshot]" id="boilerplate_findings_attributes_1444981219987_screenshot">    </textarea>
```

**Comment by nathanvda** [comment]
I am assuming you are using haml? Check this: http://haml.info/docs/yardoc/file.FAQ.html#q-preserve

**Comment by arta** [comment]
Thanks, but how can the `Haml::Helpers#find_and_preserve` help with dynamically generated nested textareas (differentiated with some sort of dynamically generated child_index)?

**Comment by nathanvda** [comment]
Are you asking me for a technical explanation why it works, or are you stating that it does not help in your case? In that case it is maybe best to open a new issue? 

Please note that the view is render server-side (only once), so it is unclear to me what you mean when you say "differentiated with a dynamically generated child-index", so imho you will be using javascript/callbacks to add that index, and in that case haml is no longer responsible for the added whitspace imho?

**Comment by arta** [comment]
I may be confused, but It seems to me that `Haml::Helpers#find_and_preserve` is not a solution to the issue #323. Hence my question. Thanks.

**Comment by arta** [comment]
`Haml::Template.options[:ugly] = true` or `Haml::Template.options[:remove_whitespace] = true` in config/initializers/haml.rb fixes it. 

See https://github.com/haml/haml/issues/643.
